packages/web-components/src/components/ic-chip/ic-chip.css

```diff
@@ -0,0 +1,142 @@
+@import "../../global/normalise.css";
+
+.chip {
+  padding: var(--ic-space-xxs);
+  font-size: 14px;
+  border-radius: 80px;
+  display: inline-block;
+  text-align: center;
+  text-decoration: none;
+}
+
+.label {
+  padding: 0 var(--ic-space-xs);
+}
+
+.chip.dismissible .label {
+  padding-right: var(--ic-space-xxs);
+}
+
+.chip.dismissible.small .label {
+  padding-right: 6px;
+}
+
+.chip.dismissible.large .label {
+  padding-right: var(--ic-space-xxxs);
+}
+
+.chip.dismissible {
+  padding: 0;
+}
+
+.chip.small {
+  padding: var(--ic-space-xxxs);
+}
+
+.chip.dismissible.small {
+  padding-right: 0;
+}
+
+.chip.dismissible.large {
+  padding: var(--ic-space-xxs);
+}
+
+.chip.large {
+  padding: var(--ic-space-xs);
+}
+
+.chip.dismissible .container {
+  justify-content: space-between;
+}
+
+.container {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.chip.clickable:focus,
+.chip.clickable.focussed {
+  background-color: var(--ic-architectural-300);
+  box-shadow: var(--ic-border-focus);
+  outline: var(--ic-hc-focus-outline);
+}
+
+.filled {
+  background-color: var(--ic-architectural-200);
+  color: black;
+  border: none;
+}
+
+.filled.disabled {
+  background-color: var(--ic-architectural-100);
+}
+
+.filled.clickable:hover {
+  background-color: var(--ic-architectural-300);
+  cursor: pointer;
+}
+
+.filled.clickable:active {
+  background-color: var(--ic-architectural-400);
+}
+
+.outline {
```

Comment:
Looks like there's a background colour on the outline variant:
![image](https://user-images.githubusercontent.com/111882109/208088704-aad2f82a-0522-4d5d-a69a-65ab7f539d35.png)
but it's just transparent in the designs (and this is making the change to the active / pressed colour when it's clickable less noticeable)

Also the hover state on the clickable variant doesn't seem to work for me? Maybe it's just not showing up because of the background